src/Facade/Services/SalesArticleService.cs

```diff
@@ -22,6 +28,32 @@ public class SalesArticleService : FacadeService<SalesArticle, string, SalesArti
             : base(repository, transactionManager)
         {
             this.coreTypeRepository = coreTypeRepository;
+            this.salesArticleRepository = repository;
+            this.transactionManager = transactionManager;
+        }
+
+        public IResult<ResponseModel<SalesArticlesReallocator>> Reallocate(string oldTariffId, string newTariffId, IEnumerable<string> privileges)
+        {
+            var oldId = int.Parse(oldTariffId);
+            var newId = int.Parse(newTariffId);
+
+            try
+            {
+                var articlesForReallocation = this.salesArticleRepository.FindAll().Where(x => x.TariffId == oldId);
+
+                foreach (var salesArticle in articlesForReallocation)
+                {
+                    salesArticle.TariffId = newId;
```

Comment:
cool but testable logic so should be domain code

Reply:
Ahhh yeah sorry I forgot we don't put logic like that in facades, was just trying to get that into my head before furlough. Will move it